thirdparty/scripts/build_arrow.sh

```diff
@@ -109,6 +109,22 @@ build_arrow() {
   make VERBOSE=1 -j$PARALLEL
   make install
 
+  cmake -DCMAKE_BUILD_TYPE=Release \
```

Comment:
why run `cmake` a second time?

Reply:
That was an accidental commit, I'm removing it